source/sourcing_and_processing_data/adding_processors_to_a_dataset.rst

```diff
@@ -0,0 +1,300 @@
+Adding processors to a dataset
+==============================
+
+Processors are tools that can be added to a dataset in order to modify, improve or enrich its data. In the OpenDataSoft platform, processors are classified in 4 different categories:
+
+* Processors for geographical mapping
```

Comment:
I am not confortable with 'mapping' and 'transformations', processors in these categories perform more global operations. Could we find a global wording ?

Reply:
Those are the terms used in the platform, that's why I chose to use them in the documentation. But as soon as the names of the processors categories change in the platform, I will modify the documentation as well :)